docs/static/includes/interfaces/int.pe.schema.tf

```diff
@@ -103,10 +128,11 @@ resource "azurerm_private_endpoint_application_security_group_association" "this
   application_security_group_id = each.value.asg_resource_id
 }
 
+# You need an additional resource when not managing private_dns_zone_group with this module:
 
-
+# In your output you need to select the correct resource based on the value of var.private_endpoints_manage_dns_zone_group:
 output "private_endpoints" {
-  value       = azurerm_private_endpoint.this
+  value       = var.private_endpoints_manage_dns_zone_group ? azurerm_private_endpoint.this_managed_dns_zone_groups : azurerm_private_endpoint.this_unmanaged_dns_zone_groups
```

Comment:
If the above comment about the map having the control flag is accepted this would also mean that the two different internal maps of `azurerm_private_endpoint.this_managed_dns_zone_groups` and `azurerm_private_endpoint.this_unmanaged_dns_zone_groups` would need to be merged back together for output.

Reply:
Agreed. Let's wait to see what the consensus is